cryptography/requirements.txt

```diff
@@ -0,0 +1 @@
+bcrypt~=3.2.0
```

Comment:
Since this PR depends pretty explicitly on a lot of FastAPI code, shouldn't it declare that in it's requirements somehow?

Reply:
Oh I see that you have different requirements for different folders in here.

...does that work with Python packaging?

Reply:
technically you could make it work by concatenating them in the `setup.py` file

Reply:
but its not quite standard, no

Reply:
There are for sure more changes needed to better support piecewise imports of each `fideslib` directory, which should all behave as standalone importable packages. I don't believe this code should be merged until we determine a good strategy to manage that.